### PR TITLE
Minor changes in the documentation

### PR DESCRIPTION
**Pull request by hoijui**

### CODE REVIEW

File: doc/caveats.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ equivalent to creating a new commit with the edited content and discarding the
 old one. Hence, if a reply already exists on the old commit, it will not be
 removed and the reply will still refer to the old commit. However, we do not
 consider this restriction sever, especially when compared to mailing-lists,
-which inhibit the same restriction. Note that we currently still lack an tool
+which inhibit the same restriction. Note that we currently still lack a tool
```
Comment (neithernut) [inline doc/caveats.md]: Not sure whether this is correct. However, I'm inclined to trust you on this.

File: doc/caveats.md [hunk 2]
```diff
@@ -28,12 +28,11 @@ cross-platform notification of events in git repositories (e.g. pushes).
...
-issues and messages. This is, however unrelated and still work in progress.
+issues and messages. This is however, unrelated and still work in progress.
 Additionally, it would still require reporters of issues to expose them via a
 public repository.
 
 It was also suggested that maintainers or service providers could provide
 special public bug repositories with public push access. Those repositories
 would have specially crafted hooks installed, implementing some sort of access
 control, e.g. preventing issues from being deleted by unauthorized actors.
-
```
Comment (neithernut) [inline doc/caveats.md]: I like keeping a newline at the end of text and source files. In my experience, it helps with merges in some situations.

File: doc/caveats.md [hunk 3]
```diff
@@ -28,12 +28,11 @@ cross-platform notification of events in git repositories (e.g. pushes).
 Using such a mechanism, contributors could notify a project maintainer about
 changes in their public repositories, including issues or messages, via a public
 API. A maintainer could set up automated fetches of new issues, importing new
-issues and messages. This is, however unrelated and still work in progress.
+issues and messages. This is however, unrelated and still work in progress.
```
Comment (neithernut) [inline doc/caveats.md]: My intuition tells me to put a comma before and after the "however".

**Comment by neithernut** [comment]
Thanks for your contribution.

File: doc/caveats.md (hunk 1 above)
Comment (hoijui) [reply doc/caveats.md]: am sure about it, but... the best poof I could find is this:

Google finds:

* "an tool": 833.000
* "a tool": 114.000.000

File: doc/caveats.md (hunk 2 above)
Comment (hoijui) [reply doc/caveats.md]: ouh yes I noticed when opening the other files, then forgot to undo this, sorry!
will change

File: doc/caveats.md (hunk 3 above)
Comment (hoijui) [reply doc/caveats.md]: hmm... it starts to get complicated then..
What if we take the "however" to the front of the sentence?

File: doc/caveats.md (hunk 2 above)
Comment (hoijui) [reply doc/caveats.md]: done.

File: doc/caveats.md (hunk 3 above)
Comment (hoijui) [reply doc/caveats.md]: did that.

File: doc/caveats.md (hunk 1 above)
Comment (neithernut) [reply doc/caveats.md]: Ah. Right. Me was simply too stupid to read the diff correctly.

Comment (hoijui) [reply doc/caveats.md]: ahhh :D

**Pull request merged**